cartographer_rviz/cartographer_rviz/drawable_submap.h

```diff
@@ -62,19 +65,25 @@ class DrawableSubmap : public QObject {
   bool MaybeFetchTexture(ros::ServiceClient* client);
 
   // Returns whether an RPC is in progress.
```

Comment:
Actually, it behaves the same as before -  since the rendering method DrawableSubmaps:UpdateSceneNode() is the one which sets query_in_progress_=false, and it is guarded by a mutex until rendering is finished, this effectively behaved like QueryOrRenderInProgress() even when it was called QueryInProgress. 

My motivation for this was to preserve the original behaviour. I had to split this flag into two separate flags for querying and rendering (it turned out that I need to wait only for query to end in the destructor; rendering never happened after the destructor was called, so I would wait indefinitely).

Will update the comment.